timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -36,12 +36,18 @@ types:
   definitions:
     default-package: com.palantir.atlasdb.timelock.api
     objects:
+      ConjureIdentifiedVersion:
+        fields:
+          id: uuid
+          version: safelong
```

Comment:
since we'd already declared the external java `Long` I'd suggest using it? I get that this usage is probably safe, but probably best to be consistent especially with places where we really don't want this to be `safelong` (i.e. timestamps)

Reply:
Done